ee/fleetd-chrome/src/db.ts

```diff
@@ -4,6 +4,7 @@ import * as SQLite from "wa-sqlite";
 // Alphabetical order
 import Table from "./tables/Table";
 import TableChromeExtensions from "./tables/chrome_extensions";
+import TableDiskInfo from "./tables/disk_info";
```

Comment:
I'm missing line 25 when I pull down this branch

`VirtualDatabase.register(sqlite3, db, new TableDiskInfo(sqlite3, db));`